src/lib/crypto/kimchi_backend/gadgets/bitwise.mli

```diff
@@ -2,18 +2,48 @@
 type rot_mode = Left | Right
 
 (** 64-bit rotation of rot_bits to the `mode` side
- *  - word of maximum 64 bits to be rotated
+ * - check64: whether to check the input word is at most 64 bits (default is false)
+ * - word of maximum 64 bits to be rotated
  * - rot_bits: number of bits to be rotated
  * - mode: Left or Right
  * Returns rotated word
  *)
 val rot64 :
      (module Snarky_backendless.Snark_intf.Run with type field = 'f)
+  -> ?check64:bool (* false *)
   -> 'f Snarky_backendless.Cvar.t
   -> int
   -> rot_mode
   -> 'f Snarky_backendless.Cvar.t
 
+(** 64-bit bitwise logical shift left of bits to the `mode` side
+ * Inputs
+ *  - check64: whether to check the input word is at most 64 bits (default is false)
+ *  - word of maximum 64 bits to be shifted
+ *  - bits: number of bits to be shifted
+ * Output: left shifted word (with bits 0s at the least significant positions)
+ *)
```

Comment:
Nit: use the ocamldoc syntax (see https://v2.ocaml.org/manual/ocamldoc.html#ss:ocamldoc-tags)
```suggestion
(** 64-bit bitwise logical shift left of bits to the `mode` side
 * 
 *  @param check64  whether to check the input word is at most 64 bits (default is false)
 *  @param word  word of maximum 64 bits to be shifted
 *  @param bits  number of bits to be shifted
 *  @return left shifted word (with bits 0s at the least significant positions)
 *)
```